src/shared/codelens/pythonCodeLensProvider.ts

```diff
@@ -149,32 +151,43 @@ def ${debugHandlerFunctionName}(event, context):
     }
 }
 
-const fixFilePathCapitalization = (filePath: string): string => {
+export function getLocalRootVariants(filePath: string): string[] {
     if (process.platform === 'win32') {
-        // TODO : TEMP : https://github.com/aws/aws-toolkit-vscode/issues/484
-        return filePath.replace(/^[a-z]/, match => match.toLowerCase())
+        if (PATH_STARTS_WITH_DRIVE_LETTER_REGEX.test(filePath)) {
```

Comment:
This condition is redundant. If it doesn't match, then `filePath.replace` will no-op.

Reply:
If the path is not a local windows path, it should be returned as-is (eg: //someshare/src/x.js )  - there are tests to validate this input too.